### ISSUE TITLE
Show selected categories at the top when editing entry

### ISSUE THREAD

**Issue by shbchk** [ExpressionEngine/ExpressionEngine#653]
Please make Categories UI in CP more like new Relationships field - it's great!

![Current UI not so great](http://shbchk.tmweb.ru/i/ee.png)

**Comment by shbchk** [comment]
To replace broken image above:

It is impossible to tell which countries is checked unless you scroll down the entire list
![screenshot-2021 01 17-17_54_10](https://user-images.githubusercontent.com/12234507/104846816-7ea26f00-58ed-11eb-9884-504583483d2d.png)

**Comment by shbchk** [comment]
:'(

**Comment by robsonsobral** [comment]
Couldn't we alter this component to move the selected options to the top of the list?

**Comment by shbchk** [comment]
Still nothing on this? I've seen [this PR](https://github.com/ExpressionEngine/ExpressionEngine/pull/1526), but I do not see anything new in my CP

**Comment by shbchk** [comment]
Can it be moved to the top of to-do list? Pretty pleeeaase :)

**Comment by Yulyaswan** [comment]
working on it

**Comment by shbchk** [comment]
So, if I got it right, this was reverted back to as it was? Is there a chance of having a friendly UX when it comes to editing long lists of categories? 

If there are issues with visible order, maybe somehow show selected categories apart from the list itself, I mean without mutating the original list? 

Like this:
![27251010-164a-4e91-9394-cfd5a2de1cc4](https://user-images.githubusercontent.com/12234507/215799660-9c5f0bdd-345e-4845-a918-67663c5911f0.png)
